cwf/gateway/deploy/roles/ovs/tasks/debian.yml

```diff
@@ -28,10 +28,10 @@
       - dh-autoreconf
       - libssl-dev
       - libtool
+      - net-tools
```

Comment:
just curious: how did you find this dependency?

Reply:
net-tools contains ifconfig - ifconfig is used in one of the ansible steps